content/numpy/concepts/ndarray/terms/reshape/reshape.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.reshape()'
```

Comment:
confused as to why there are two entries for reshape. One under built-in functions and another under ndarray

Reply:
One's a function, one's a method.